testcomponent/features/updateMany.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Update Many Records
+
+  Background:
+    Given I have inserted these Records
+            """
+            [
+                {
+                    "id": 1,
+                    "name": "Tester 1",
+                    "age": "TestAge1"
+                },
+                {
+                    "id": 2,
+                    "name": "Tester 2",
+                    "age": "TestAge2"
+                },
+                {
+                    "id": 3,
+                    "name": "Tester 2",
+                    "age": "TestAge3"
+                }
+            ]
+            """
+
+  Scenario: Update One Record
+    When I update records with name "Tester 1" age to "Testing Age"
+    Then there are 1 matched, 1 modified, 0 upserted records
+    And I filter on all records
```

Comment:
This `And` isn't really an assertion

Reply:
IMO that's an anti-pattern but I'm happy to go with your gut if you feel that's correct gherkin